client/layout/masterbar/logged-in.jsx

```diff
@@ -18,15 +18,15 @@ import { preload } from 'sections-preload';
 import ResumeEditing from 'my-sites/resume-editing';
 import { setNextLayoutFocus } from 'state/ui/layout-focus/actions';
 import { getSelectedSiteId } from 'state/ui/selectors';
-import { getSiteSlug, getSiteBySlug } from 'state/sites/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
 import { getStatsPathForTab } from 'lib/route/path';
 import { getCurrentUser } from 'state/current-user/selectors';
 import isDomainOnlySite from 'state/selectors/is-domain-only-site';
-import { domainManagementEdit } from 'my-sites/upgrades/paths';
+import { domainManagementList } from 'my-sites/upgrades/paths';
 
 const MasterbarLoggedIn = React.createClass( {
 	propTypes: {
-		isDomainOnlySite: React.PropTypes.bool,
+		domainOnlySite: React.PropTypes.bool,
```

Comment:
I had to rename this prop to avoid variable shadowing in `render()` and `connect()`.